ui/open-api-ui/src/test/java/io/smallrye/openapi/ui/IndexCreatorTest.java

```diff
@@ -24,13 +25,15 @@ void testCreateDefault() throws IOException {
 
         String s = new String(indexHtml);
 
-        assertTrue(s.contains("<title>SmallRye OpenAPI UI</title>"));
-        assertTrue(s.contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"theme-feeling-blue.css\" >"));
-        assertTrue(s.contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"style.css\" >"));
-        assertTrue(s.contains("url: '/openapi',"));
-        assertTrue(s.contains("<img src='logo.png' alt='SmallRye OpenAPI UI'"));
-        assertTrue(s.contains("dom_id: '#swagger-ui',"));
-        assertTrue(s.contains("deepLinking: true,"));
+        assertThat(s).contains("<title>SmallRye OpenAPI UI</title>");
+        assertThat(s).contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"theme-feeling-blue.css\" >");
+        assertThat(s).contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"style.css\" >");
+        assertThat(s).contains("url: '/openapi',");
+        assertThat(s).contains("<img src='logo.png' alt='SmallRye OpenAPI UI'");
+        assertThat(s).contains("dom_id: '#swagger-ui',");
+        assertThat(s).contains("deepLinking: true,");
+        assertThat(s).contains("queryConfigEnabled: false");
```

Comment:
When using the default (from swagger ui) we just omit the value. (Rather than adding it with the default)

Reply:
I will change to assertThat(s).contains("queryConfigEnabled"); its ok?

Reply:
Yea